app/controllers/profiles_controller.rb

```diff
@@ -7,13 +7,29 @@ def index
   end
 
   def show
-    profile = Profile.find(params[:id])
     authorize profile
     render json: ProfileSerializer.new(profile)
   end
 
+  def tailoring_file
+    return unless profile.tailored?
+
+    send_data XccdfTailoringFile.new(
```

Comment:
Just a thought, maybe we could include a link to this endpoint in the regular `show` endpoint - there's info on how to do this on https://github.com/Netflix/fast_jsonapi (doesn't need to be in this PR)

Reply:
Seems useful. I'll create a card to track it: https://projects.engineering.redhat.com/browse/RHICOMPL-509